onnxruntime/core/framework/allocatormgr.h

```diff
@@ -6,11 +6,17 @@
 #include "core/common/common.h"
 #include "core/framework/allocator.h"
 #include "core/session/onnxruntime_c_api.h"
+#include <map>
 
 namespace onnxruntime {
 
 using AllocatorFactory = std::function<std::unique_ptr<IAllocator>(OrtDevice::DeviceId)>;
 
+using AllocatorMap = std::map<int, AllocatorPtr>;
+using MemoryInfoSet = std::set<OrtMemoryInfo>;
+
+const int CPU_ALLOCATOR_DEVICE_ID = 0;
```

Comment:
>const int CPU_ALLOCATOR_DEVICE_ID = 0 [](start = 0, length = 37)

Should we call it DEFAULT_CPU_ALLOCATOR_DEVICE_ID? We might have a separate id for NUMA CPUs later.